graphql/playground/apollo_sandbox_playground.go

```diff
@@ -0,0 +1,57 @@
+package playground
+
+import (
+	"html/template"
+	"net/http"
+)
+
+var apolloSandboxPage = template.Must(template.New("ApolloSandbox").Parse(`<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>{{.title}}</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <link rel="icon" href="https://embeddable-sandbox.cdn.apollographql.com/_latest/public/assets/favicon-dark.png">
+	<style>
+	body {
+		margin: 0;
+		overflow: hidden;
+	}
+</style>
+</head>
+
+<body>
+  <div style="width: 100vw; height: 100vh;" id='embedded-sandbox'></div>
+  <!-- NOTE: New version available at https://embeddable-sandbox.cdn.apollographql.com/ -->
+  <script rel="preload" as="script" crossorigin="anonymous" integrity="{{.mainSRI}}" type="text/javascript" src="https://embeddable-sandbox.cdn.apollographql.com/58165cf7452dbad480c7cb85e7acba085b3bac1d/embeddable-sandbox.umd.production.min.js"></script>
+  <script>
+{{- if .endpointIsAbsolute}}
+	const url = {{.endpoint}};
+{{- else}}
+	const url = location.protocol + '//' + location.host + {{.endpoint}};
+{{- end}}
+  new window.EmbeddedSandbox({
+    target: '#embedded-sandbox',
+    initialEndpoint: url,
+		persistExplorerState: true,
```

Comment:
It would also be nice to have a link to the options documentation like https://www.apollographql.com/docs/graphos/explorer/sandbox/#initialendpoint (or possibly somewhere else)

Reply:
Thanks, I will fix initial value and extend the arguments of the handler to allow passing options and comment the description.